app/lib/widgets/wallets/wallet_card.dart

```diff
@@ -161,11 +164,42 @@ class _WalletCardWidgetState extends ConsumerState<WalletCardWidget> {
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
-              Text(
-                widget.wallet.name,
-                style: Theme.of(context).textTheme.titleLarge!.copyWith(
-                      color: Theme.of(context).colorScheme.onSecondaryContainer,
+              Row(
+                children: [
+                  Text(
+                    widget.wallet.name,
+                    style: Theme.of(context).textTheme.titleLarge!.copyWith(
+                          color: Theme.of(context)
+                              .colorScheme
+                              .onSecondaryContainer,
+                        ),
+                  ),
+                  const Spacer(),
+                  Container(
+                    padding:
+                        const EdgeInsets.symmetric(horizontal: 12, vertical: 6),
+                    decoration: BoxDecoration(
+                      border: Border.all(
+                        color: wallet!.verificationStatus ==
+                                VerificationState.VERIFIED.name
+                            ? Theme.of(context).colorScheme.primary
+                            : Theme.of(context).colorScheme.error,
+                      ),
+                      borderRadius: BorderRadius.circular(20),
                     ),
+                    child: Text(
+                        wallet.verificationStatus.toLowerCase().replaceFirst(
+                              wallet.verificationStatus[0].toLowerCase(),
+                              wallet.verificationStatus[0].toUpperCase(),
+                            ),

```

Comment:
we might need the capitalize method in some places, so it would be nice if we have a method for it in helpers
```suggestion
                        wallet.verificationStatus[0].toUpperCase() + wallet.verificationStatus.substring(1).toLowerCase()
```